### PR TITLE
Bug 2115037: Text correction in DR policy creation

### PR DESCRIPTION
**Pull request by TimothyAsirJeyasing**
Signed-off-by: Timothy Asir <tjeyasin@redhat.com>

### CODE REVIEW

**Comment by TimothyAsirJeyasing** [comment]
@bipuladh , @SanjalKatiyar Please review

**Comment by SanjalKatiyar** [comment]
/retitle Bug 2115037: Text correction in DR policy creation

File: locales/en/plugin__odf-console.json [hunk 1]
```diff
@@ -27,7 +27,7 @@
   "Create DRPolicy": "Create DRPolicy",
   "Get a quick recovery in a remote or secondary cluster with a disaster recovery (DR) policy": "Get a quick recovery in a remote or secondary cluster with a disaster recovery (DR) policy",
   "Connect clusters": "Connect clusters",
-  "Enables mirroring/replication between the two selected clusters and ensures failover from primary cluster to secondary cluster in the event of an outage.": "Enables mirroring/replication between the two selected clusters and ensures failover from primary cluster to secondary cluster in the event of an outage.",
+  "Enables mirroring/replication between the two selected clusters and ensures failover or relocate from primary cluster to secondary cluster in the event of an outage or planned maintenance.": "Enables mirroring/replication between the two selected clusters and ensures failover or relocate from primary cluster to secondary cluster in the event of an outage or planned maintenance.",
```
Comment (GowthamShanmugam) [inline locales/en/plugin__odf-console.json:30]: Is UX is already reviewed this?

Comment (SanjalKatiyar) [reply locales/en/plugin__odf-console.json:30]: @TimothyAsirJeyasing please ping Chandan once and confirm with him...

Comment (TimothyAsirJeyasing) [reply locales/en/plugin__odf-console.json:30]: chandkum@redhat.com Please review

**Review by Chandan141996 — CHANGES_REQUESTED** [review]
@TimothyAsirJeyasing @GowthamShanmugam 
I would suggest changing the description text to: 
Enables mirroring/replication between two selected clusters, ensuring failover or relocation between the two clusters in the event of an outage or planned maintenance.

**Review by GowthamShanmugam — APPROVED** [review]

**Pull request merged**